src/SqlBinding/SqlBindingUtilities.cs

```diff
@@ -173,8 +180,8 @@ public static void GetTableAndSchema(string fullName, out string schema, out str
 
             if (pieces.Length == 2)
             {
-                schema = $"'{pieces[0]}'";
-                tableName = pieces[1];
+                quotedSchema = $"'{pieces[0].Replace("'", "''")}'";
```

Comment:
1. Would suggest adding a comment here too about the replacement
2. Or probably useful to just make a helper function to do the escaping (reduce change of typos and the name makes it clearer what it's doing)
3. You can also just escape the fullName once at the beginning instead of escaping 3 separate times.